docs/aws-config.md

```diff
@@ -151,6 +151,10 @@ Specify the following values in the example files:
 
     * Replace `<YOUR_STORAGE_CLASS_NAME>` with `gp2`. This is AWS's default `StorageClass` name.
 
+* (Optional) If you have multiple clusters and you want to support migration of resources between them, in file `examples/aws/10-deployment.yaml`:
+
+    * Uncomment the environment variable `AWS_CLUSTER_NAME` and replace `<YOUR_CLUSTER_NAME>` with the current cluster's name. When restoring backup, it will make Ark (and cluster it's running on) claim ownership of AWS volumes created from snapshots taken on different cluster.
```

Comment:
could you add another sentence and possibly a sample `kubectl` command explaining how to get the current cluster's name?

Reply:
I'm pretty sure the only way to get this information is to go to EC2 and look at the tags applied to the cluster's instances. The kubectl doesn't is not guaranteed to be the right name.

Reply:
Ah, ok - it would be really useful to either describe that or include an `aws` CLI cmd to fetch it

Reply:
I agree that the `kubectl` context is unreliable, even though the chances of getting right name from it are still high I guess if it was configured according to deployment tool instructions. I changed it to short listing of how to get the name-related Tags from the instances using aws CLI